### PR TITLE
Add support for XRP sequence when converting an Operation into a RippleLikeTransactionApi.

### PR DESCRIPTION
**Pull request by phaazon**
# Content

That should fix the `nullptr` exception @meriadec saw.

Relates to LLC-349.

# Weird picture of an animal

### CODE REVIEW

File: core/src/database/migrations.cpp [hunk 1]
```diff
@@ -408,7 +408,8 @@ namespace ledger {
                     "sender VARCHAR(255) NOT NULL,"
                     "receiver VARCHAR(255) NOT NULL,"
                     "fees VARCHAR(255) NOT NULL,"
-                    "confirmations BIGINT NOT NULL"
+                    "confirmations BIGINT NOT NULL",
+                    "sequence BIGINT NOT NULL",
```
Comment (phaazon) [inline core/src/database/migrations.cpp]: WHOOOOOOOOOPS, need to remove that.

**Comment by phaazon** [comment]
> Maybe add a test to see if it parses right, getSequence returns a non null pointer

Oh yes I forgot. On my way.
